.github/actions/acceptance-test-action/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Acceptance Test Action"
+description: "Run acceptance tests for a resource"
+author: "MongoDB"
+inputs:
+  forceTestAcc:
+    description: "if unconditionally trigger tests"
+    required: true
+  filters:
+    description: "files to detect if tests should be run"
+    required: true
+  testRegex:
+    description: "regular expression for test methods"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: ${{ inputs.filters }}
```

Comment:
Something to keep in mind with this approach is that we call `dorny/paths-filter@v2` for every job. Before we were calling `dorny/paths-filter@v2` once, and then use the result of that step to run the correct jobs
![Screenshot 2023-11-13 at 12 17 12](https://github.com/mongodb/terraform-provider-mongodbatlas/assets/5663078/5e50a122-cd2f-4821-9411-a2d5a53a28fa).

**One cons of this approach is that we will always create a run job to run every acceptance test, the job will then terminate once `dorny/paths-filter@v2` is run.** Right now the job is skipped instead. 

I would suggest you to add pros/cons of this approach in the description of your PR to help the discussion


(not strong opinion here I just want to highlight this new behaviour)

Reply:
you're right the jobs will be created, but the tests steps will be skipped if not need to run so at the end I think it'll be equivalent.   Also it's true that we call path-filters multiple times but I suppose this action internally has to do the code for each filter group so at then end it does a similar job.

one pro of current version is that it's easier to see what acceptance tests were skipped.

i can change the description to add pros and cons